PBM-883-Update intro page (#36)
percona/pbm-docs: source/intro.rst

The following example illustrates how to use |PBM|.

With |PBM| up and running in your environment, make a backup:

.. code-block:: bash

   $ pbm backup

> To also save all events that occurred to the data between the backups, enable incremental backups:

Author: Fixed

> To also save all events that occurred to the data between the backups, enable saving oplog slices:

Reviewer: ```suggestion
To also save all events that occurred to the data while the backup was running, enable saving oplog slices:
```

Author: Oplog that relates to the backup snapshot is saved together with that snapshot. This is about oplog events that occurred between the snapshots.

Reviewer: Ah, I see. Then it might make sense to explain this behavior in the paragraph above that explains the `pbm backup` command?

   .. code-block:: text

      Backup snapshots:
        2021-02-03T08:08:15Z [complete: 2021-02-03T08:08:35Z]
        2021-02-05T13:55:55Z [complete: 2021-02-05T13:56:15Z]
        2021-02-07T13:57:58Z [complete: 2021-02-07T13:58:17Z]
        2021-02-09T14:06:06Z [complete: 2021-02-09T14:06:26Z]
        2021-02-11T14:22:41Z [complete: 2021-02-11T14:23:01Z]

      PITR <on>:
       2021-02-03T08:08:36Z-2021-02-09T12:20:23Z
       2021-02-09T14:06:27Z-2021-02-11T14:22:40Z

> The output lists the valid time ranges for the restore. The desired time (February 7th, 03:00 UTC) falls within the ``2021-02-03T08:08:36Z-2021-02-09T12:20:23Z`` range, so let’s restore the database up to that time. 

Reviewer: ```suggestion
The output lists the valid time ranges for the restore. The desired time (February 7th, 03:00 UTC) falls within the ``2021-02-03T08:08:36Z-2021-02-09T12:20:23Z`` range, so let’s restore the state of database back to that time. 
```

> Since the restore and saving oplog slices are exclusive operations and cannot  run together, let’s stop oplog slicing first:

Reviewer 2: ```suggestion
Since the restore and saving oplog slices are exclusive operations and cannot run together, let’s stop oplog slicing first:
```
